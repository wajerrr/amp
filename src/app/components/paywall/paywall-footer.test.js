import React from 'react';
import renderer from 'react-test-renderer';
import PaywallFooter from './paywall-footer';

describe('PaywallFooter', () => {
  it('should match the snapshot', () => {
    const paywallFooter = renderer
      .create(
        <PaywallFooter
          data={{
            id: 'regwall-footer',
            type: 'footer',
            content: {
              text: ['Test footer'],
            },
          }}
        />
      )
      .toJSON();
    expect(paywallFooter).toMatchSnapshot();
  });
});
