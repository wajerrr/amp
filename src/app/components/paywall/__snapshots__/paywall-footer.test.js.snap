// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PaywallFooter should match the snapshot 1`] = `
<footer
  className={undefined}
>
  Test footer
</footer>
`;
